docs/standard/serialization/system-text-json/polymorphism.md

```diff
@@ -20,7 +21,9 @@ In this article, you will learn how to serialize properties of derived classes w
 
 ## Serialize properties of derived classes
 
-Serialization of a polymorphic type hierarchy is _not_ supported. For example, if a property is defined as an interface or an abstract class, only the properties defined on the interface or abstract class are serialized, even if the runtime type has additional properties. The exceptions to this behavior are explained in this section.
+:::zone pivot="dotnet-core-3-1,dotnet-5-0,dotnet-6-0"
+
+Prior to .NET 7, `System.Text.Json` _did_n't support_ the serialization of polymorphic type hierarchies. For example, if a property is defined as an interface or an abstract class, only the properties defined on the interface or abstract class are serialized, even if the runtime type has additional properties. The exceptions to this behavior are explained in this section.
```

Comment:
Would it make sense to link to the new article here as well?

Reply:
Good idea, added.